contrib/node/tests/node/node-thrift-js-test/yarn.lock

```diff
@@ -0,0 +1,2027 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+abbrev@1:
+  version "1.1.1"
+  resolved "https://artifactory.twitter.biz:443/api/npm/js-virtual/abbrev/-/abbrev-1.1.1.tgz#f8f2c887ad10bf67f634f005b6987fed3179aac8"
```

Comment:
@stuhood, this one's going to be a blocker. I'm sorry I didn't catch this one sooner.

@itsmesatwik: you'll want to remove twitter-specific details from your yarn.lock. Please re-hydrate this dependency using the yarn registry.

Reply:
Fixed